test/export_test.cc

```diff
@@ -0,0 +1,57 @@
+#include "cell.h"
```

Comment:
to be consistent I would recommend to rename it to `exporter_test.cc`
in general: `foo_bar.h` -> `foo_bar_test.cc`